tests/containers/podman_image.pm

```diff
@@ -28,8 +29,8 @@ sub run {
     allow_selected_insecure_registries(runtime => $runtime);
     for my $iname (@{$image_names}) {
         test_container_image(image => $iname, runtime => $runtime);
-        build_container_image(image => $iname, runtime => $runtime);
-        if (check_os_release('suse', 'PRETTY_NAME')) {
+        build_container_image(image => $iname, runtime => $runtime) if is_sle_container_supported;
+        if (check_os_release('suse', 'PRETTY_NAME') && is_sle_container_supported) {
```

Comment:
This by the way means that `if(is_sle_container_supported()){test_opensuse_based_image()};` which does not make a sense.